src/components/Note.vue

```diff
@@ -218,6 +219,12 @@ export default {
 						showError(t('notes', 'Error from Nextcloud server: {msg}', { msg: note.errorType }))
 					}
 					this.startRefreshTimer()
+
+					const newContent = this.$route.query.new
+					if (this.isNewNote && (newContent !== '' && newContent != null)) {
+						this.note.content = newContent
+						this.onManualSave()
+					}
```

Comment:
I don't like this approach, because it generates a very long URI. Instead, you should add a second parameter to `createNote` that takes the content and directly sends it to the server in the same request that creates the new note.